pkg/api/v1/types.go

```diff
@@ -1952,14 +1952,30 @@ type PodLogOptions struct {
 
 	// The container for which to stream logs. Defaults to only container if there is one container in the pod.
 	Container string `json:"container,omitempty"`
-
-	// Follow the log stream of the pod.
-	// Defaults to false.
+	// Follow the log stream of the pod. Defaults to false.
 	Follow bool `json:"follow,omitempty"`
-
-	// Return previous terminated container logs.
-	// Defaults to false.
+	// Return previous terminated container logs. Defaults to false.
 	Previous bool `json:"previous,omitempty"`
+	// A relative time in seconds before the current time from which to show logs. If this value
+	// precedes the time a pod was started, only logs since the pod start will be returned.
+	// If this value is in the future, no logs will be returned.
```

Comment:
The time the client thinks is the future and the time the user thinks is the future and the time the node and container think is the future can all differ.